admin-dev/themes/new-theme/js/app/pages/localization/components/currencies/currencies-form.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <form>
+    <div class="row">
+      <div  class="col-sm-4">
+        <label>{{trans('label_currency')}}</label>
+        <PSHelp 
+          content="Vivamus sagittis lacus vel augue laoreet rutrum faucibus." 
```

Comment:
Is it intended to stay filled with lorem ipsum data ?

Reply:
@LittleBigDev As seen with @AlexEven content of helpers must be the same as PrestaShop 1.6, you can find it on machine shuffle

Reply:
Ok then.